### ISSUE TITLE
"Usage of object.actor is deprecated for Caffeine"

### ISSUE THREAD

**Issue by rffontenelle** [eonpatapon/gnome-shell-extension-caffeine#157]
FWIW, I notice the following message while browsing `journalctl`'s logs:

```
Usage of object.actor is deprecated for Caffeine
get@resource:///org/gnome/shell/ui/environment.js:285:29
_init@/home/rffontenelle/.local/share/gnome-shell/extensions/caffeine@patapon.info/extension.js:147:9
enable@/home/rffontenelle/.local/share/gnome-shell/extensions/caffeine@patapon.info/extension.js:316:25
_callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:160:32
loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:311:26
_loadExtensions/<@resource:///org/gnome/shell/ui/extensionSystem.js:536:18
collectFromDatadirs@resource:///org/gnome/shell/misc/fileUtils.js:27:17
_loadExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:515:19
_enableAllExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:545:18
_sessionUpdated@resource:///org/gnome/shell/ui/extensionSystem.js:576:18
init@resource:///org/gnome/shell/ui/extensionSystem.js:54:14
_initializeUI@resource:///org/gnome/shell/ui/main.js:249:22
start@resource:///org/gnome/shell/ui/main.js:138:5
@<main>:1:47
```

Environment: Arch Linux 64-bit, GNOME 3.36, caffeine 35 (installed from [extensions.gnome.org](https://extensions.gnome.org/extension/517/caffeine/))

**Comment by asegarra** [comment]
I opened a pull request #159 that I think fixes the issue but seeing the announcement from maintainer it may not get fixed, you can apply the changes locally yourself if the messages bother you.

**Comment by eonpatapon** [comment]
Fixed by #159
